guide/tutorial-3-appearance-15-minutes.md

```diff
@@ -68,11 +68,12 @@ The aim of this subsection is to apply a texture on the ball. A texture on a
 rolling object can help to appreciate its movement.
 
 > **Hands on**:
-Similarly add an Appearance node to the ball. Instead of a Material node, add an
-ImageTexture node to the `texture` field of the Appearance node. Add an item to
-the `url` field using the `Add` button. Then set the value of the newly added
-`url` item to "WEBOTS\_HOME/projects/default/worlds/textures/bricks.png" using
-the file selection dialog.
+Similarly add an Appearance node to the ball. Add an ImageTexture node to the 
+`texture` field as well as a Material node to the `material` field of the 
+Appearance node. Add an item to the `url` field using the `Add` button. Then set 
+the value of the newly added `url` item to 
+"WEBOTS\_HOME/projects/default/worlds/textures/bricks.png" using the file 
```

Comment:
Is `bricks.png` correct here? (You changed it to `red_brick_wall.png` just bellow)